_data/sw_clients.yml

```diff
@@ -397,6 +430,7 @@
           - plain
     - name: The Lounge
       # ref: https://github.com/thelounge/thelounge/projects/1
+      #      https://github.com/kiwiirc/irc-framework/blob/master/docs/ircv3.md
```

Comment:
wrong link

Reply:
It's not the wrong link. The Lounge uses irc-framework.

Reply:
ok, my bad